frontend/views/components/Forms/Password.vue

```diff
@@ -14,19 +12,17 @@
       v-model='value[name]'
       @input='v[name].$touch()'
     )
-    button.is-icon(
+    .button.is-icon(
```

Comment:
This changes from `<button>` to `<span>`. Now toggling the password visibility isn't accessible by keyboard. Any reason to do it?

Reply:
Yes that avoid the flash that show the password for a brief moment when the form is unloaded. There is probably a better approach, I'm happy to hear any better suggestion

Reply:
Yes that avoid the flash that show the password for a brief moment when the form is unloaded. There is probably a better approach, I'm happy to hear any better suggestion

Reply:
Okay, I'll take a look today and let you know.

Reply:
I couldn't replicate the flash issue. Could you explain it in more detail?